src/agb/image.py

```diff
@@ -202,7 +203,7 @@ def from_file(file_path: str) -> tuple[Image, Palette]:
         reader = png.Reader(f)
         width, height, data, attributes = reader.read() # type: ignore
         attributes: dict[str, int | str] = attributes
-        assert attributes['bitdepth'] in (2, 4, 8), 'Invalid bitdepth'
+        assert math.log2(attributes['bitdepth']) % 1 == 0, 'Invalid bitdepth'
```

Comment:
This still does not support other bitdepths in the rest of the logic of this module. I will fix it :)